tests/integration/Authentication.Tests.ps1

```diff
@@ -1,6 +1,8 @@
-$path = $MyInvocation.MyCommand.Path
-$src = (Split-Path -Parent -Path $path) -ireplace '[\\/]tests[\\/]integration', '/src/'
-Get-ChildItem "$($src)/*.ps1" -Recurse | Resolve-Path | ForEach-Object { . $_ }
+BeforeAll {
+    $path = $PSCommandPath
+    $src = (Split-Path -Parent -Path $path) -ireplace '[\\/]tests[\\/]unit', '/src/'
+    Get-ChildItem "$($src)/*.ps1" -Recurse | Resolve-Path | ForEach-Object { . $_ }
+}
```

Comment:
This could be the potential cause of the failing tests, this should be:
```powershell
BeforeAll {
    $path = $MyInvocation.MyCommand.Path
    $src = (Split-Path -Parent -Path $path) -ireplace '[\\/]tests[\\/]integration', '/src/'
    Get-ChildItem "$($src)/*.ps1" -Recurse | Resolve-Path | ForEach-Object { . $_ }
}
``` 

Mostly note it's `[\\/]tests[\\/]integration` rather than `[\\/]tests[\\/]unit`. I have the same problem as you though, for me these tests pass locally 🤔

Reply:
you are right I missed it